zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -123,14 +118,6 @@ func (ob *Observer) ObserveInbound(ctx context.Context) error {
 	// https://github.com/zeta-chain/node/issues/1847
 	// TODO: move this logic in its own routine
 	// https://github.com/zeta-chain/node/issues/2204
```

Comment:
We can remove these TODOs as well

Reply:
https://github.com/zeta-chain/node/pull/2568/commits/38cb3cb68a8127c81c1baa4e0412320142dbaa43